src/utils/helpers/propCustomValidation.js

```diff
@@ -0,0 +1,8 @@
+export const checkDateString = (props, createdTime) => {
+  const date = Date.parse(props[createdTime]);
```

Comment:
why do we need a props argument? 

```suggestion
export const checkDateString = ( createdTime) => {
  const date = Date.parse(createdTime);
```

Reply:
I'm using prop-types package which required 3 arguments for custom types, where first one is props .